modin/config/pubsub.py

```diff
@@ -449,4 +452,58 @@ def add_option(cls, choice: Any) -> Any:
         raise TypeError("Cannot add a choice to a parameter where choices is None")
 
 
-__all__ = ["Parameter"]
+@contextlib.contextmanager
+def update(
+    config: Union[Parameter, dict[Parameter, Any]], value: Optional[Any] = None
+) -> Iterator[None]:
+    """
+    Set a value(s) for the specified config(s) in the scope of the context.
+
+    Parameters
+    ----------
+    config : Parameter or dict[Parameter, Any]
+        A Parameter class to set the `value` for. May be a dictionary describing multiple parameters
+        with its values.
+    value : Any, optional
+        A value to set to the config. Must be ``None`` if `config` is a dictionary.
+
+    Examples
+    --------
+    >>> class MyParameter1(Parameter, type=bool):
+    ...     default = False
+    >>> MyParameter1.get()
+    False
+    >>> with update(MyParameter1, True):
+    ...     print(MyParameter1.get()) # True
+    True
+    >>> MyParameter1.get()
+    False
+    >>> class MyParameter2(Parameter, type=bool):
+    ...     default = True
+    >>> with update({MyParameter1: True, MyParameter2: False}):
+    ...     print(MyParameter1.get()) # True
+    ...     print(MyParameter2.get()) # False
+    True
+    False
+    >>> MyParameter1.get()
+    False
+    >>> MyParameter2.get()
+    True
+    """
+    if value is None:
+        assert isinstance(config, dict)
```

Comment:
Asserts may be disabled in user code; it is better to throw `ValueError` exception.